cli/src/merge_tools/builtin.rs

```diff
@@ -918,6 +861,162 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_edit_diff_builtin_add_executable_file() {
+        let test_repo = TestRepo::init();
+        let store = test_repo.repo.store();
+
+        let added_executable_file_path = RepoPath::from_internal_string("executable_file");
+        let left_tree = testutils::create_tree(&test_repo.repo, &[]);
+        let right_tree = {
+            let store = test_repo.repo.store();
```

Comment:
looks like you can use the variable from the parent scope (i.e. delete this line)

Reply:
Done in https://github.com/jj-vcs/jj/pull/5486/commits/e1a5f583956fbaa2d771aa872fd1f5b11b27d088